### ISSUE TITLE
Validation against the complete dataset

### ISSUE THREAD

**Issue by ldesousa** [mgolshenas/Thesis#23]
The code will be very similar to what is already in the RandomForest module after [line 50](https://github.com/mgolshenas/Thesis/blob/c4493ad5a7c0747a899bc9fe98e18c93f8f3cd1b/m11_RandomForestRegression_HyperparameterTuning.py#L50):
https://github.com/mgolshenas/Thesis/blob/c4493ad5a7c0747a899bc9fe98e18c93f8f3cd1b/m11_RandomForestRegression_HyperparameterTuning.py#L50

But instead of predicting only for the test set, prediction must be made for the complete raster (1 million cells). Then the metrics functions from `sklearn` can be used again ([`r2_score`](https://scikit-learn.org/stable/modules/generated/sklearn.metrics.r2_score.html#sklearn.metrics.r2_score) and [`root_mean_squared_error`](https://scikit-learn.org/stable/modules/generated/sklearn.metrics.root_mean_squared_error.html#sklearn.metrics.root_mean_squared_error)). But this time taking as observation array the full raster (DEM01.gtif).
